### ISSUE TITLE
Wrong price - read please

### ISSUE THREAD

**Issue by stasdem** [manu354/cryptocurrency-arbitrage#22]
Hello
It is not the same as issue 18(https://github.com/manu354/cryptocurrency-arbitrage/issues/18), please read.
Generally you make great program, but still have some bugs, as I think.
1.
On markets where are no volume, spreads can be very big, and these spreads will kill all profit. 
So if we want to buy on one exchange and sold on another, we need to check not buy_price on first exchange and sell_price on second, but sell_price on first and buy_price on second. Because sell_price will be our buy price and buy_price will be our sell price.
For example:
http://joxi.ru/8AnWlgvtqy9njr
On this printscreen we are going to buy on Cryptopia for 0.0163 and sell on Bittrex for 0.0236, but when will try to buy on Cryptopia you will see big spread
http://joxi.ru/Rmz5ydWcWMLJ0r
And you will buy for the first sell_price, on this printscreen it is 0.00002292.
2.
http://joxi.ru/vAWM8Q0tkOxlZm
In my opinion you need to change this two prices, because Cryptopia have price 0.0163 and Bittrex 0.0236.
Why there are contrariwise?

Thank you.

**Comment by manu354** [comment]
See if it's fixed now. If it's not I don't really understand the problem, maybe try explaining it again? Sorry.

**Comment by gilrosenthal** [comment]
@manu354 I think this can be fixed by accepting my PR of bid/ask. I haven't had any problems with it since

**Comment by manu354** [comment]
@gilrosenthal oh shit I completely forgot about the prs. Let me get that into a dev branch.

**Comment by stasdem** [comment]
@gilrosenthal Yes, you are right. I meant this, that you use last_price, but you need to use bid and ask prices.

**Comment by stasdem** [comment]
Also I suggest to add variable volume on wich you want to trade. This variable need to be, because when you check bid and ask price you also need to check volume wich you can buy or sell for this price. 
And the right price for your volume will be summ of bid or ask prices(wich will give you your volume) divided by quantity of prices.

**Comment by chrisiew89** [comment]
Has this been fixed? I also noticed that even though the program shows that there is an arbitrage, the arbitrage is only available when you manage to buy at bid price and sell at ask price. To be able execute the arbitrage, correct way is to be able to buy at ask price and sell at bid price. Hope this explanation helps.

**Comment by roko772** [comment]
any updates?

**Comment by manu354** [comment]
I'm not working on this project anymore.

**Comment by rokcarl** [comment]
Are you planning on working on this in the future or is this the end of this repo?

**Comment by Cozy35** [comment]
Has anybody been able to rectify the ask/bid vs latest price issue?

**Comment by happyjack27** [comment]
Well yes and no.  Always sell at one and buy at the other.  In between you're talking about time to fill.


Sent via the Samsung Galaxy S7, an AT&T 4G LTE smartphone
-------- Original message --------From: Cozy35 <notifications@github.com> Date: 1/16/18  1:15 AM  (GMT-06:00) To: manu354/cryptocurrency-arbitrage <cryptocurrency-arbitrage@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [manu354/cryptocurrency-arbitrage] Wrong price - read please
  (#22) 
Has anybody been able to rectify the ask/bid vs latest price issue?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/manu354/cryptocurrency-arbitrage","title":"manu354/cryptocurrency-arbitrage","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/manu354/cryptocurrency-arbitrage"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Cozy35 in #22: Has anybody been able to rectify the ask/bid vs latest price issue?"}],"action":{"name":"View Issue","url":"https://github.com/manu354/cryptocurrency-arbitrage/issues/22#issuecomment-357872881"}}}

**Comment by Cozy35** [comment]
What I mean, since the project isn't being developed any further, has anyone attempted to fix the script from last price to either bid or ask where applicable?
